projects/plugins/jetpack/extensions/plugins/seo/index.js

```diff
@@ -208,7 +211,9 @@ const Seo = () => {
 
 			<PluginPrePublishPanel { ...jetpackSeoPublishPanelsProps }>
 				<div className="jetpack-seo-panel">
-					{ isSeoEnhancerEnabled && <SeoEnhancer disableAutoEnhance={ ! canHaveAutoEnhance } /> }
+					{ isSeoEnhancerEnabled && hasRequiredPlanForEnhancer && (
+						<SeoEnhancer disableAutoEnhance={ ! canHaveAutoEnhance } />
+					) }
```

Comment:
The check is required on the Post Publish panel as well

Reply:
Yeah, you're right, good point. I just need to make sure the "edit metadata" button would work as expected.

Reply:
oh, good point as well, we need to test this on wpcom